### DISCUSSION TITLE
How to change rate limit for auth server during local development?

### DISCUSSION THREAD

**Discussion by WesleyYue** [supabase/supabase#10326] [Questions]
While I'm developing, I might do weird things that hit the auth server in quick succession. It looks like the gotrue servers support `RATE_LIMIT_TOKEN_REFRESH` and `RATE_LIMIT_EMAIL_SENT` settings. What is the easiest way to change these while using `supabase start`? It doesn't look like the supabase config supports changing these.

**Comment by kangmingtay** [comment]
hey @WesleyYue, you can configure the `RATE_LIMIT_EMAIL_SENT` setting if you bring your own custom SMTP provider. We don't allow modifying that if you are using the default SMTP provider because that's a shared SMTP server for all our users which makes it prone to abuse.

The `RATE_LIMIT_TOKEN_REFRESH` is currently not expose in the dashboard settings but you can contact us at support if you require it to be higher for your project.

**Reply by WesleyYue** [reply]
I'm mostly interested in changing them for end-to-end tests locally. It would be great if Supabase can expose this as a setting in the `supabase start` config.
